src/main/java/fi/metatavu/dcfb/server/items/ItemController.java

```diff
@@ -99,43 +115,52 @@ public Item createItem(LocalizedEntry title, LocalizedEntry description, Categor
   public Item findItem(UUID itemId) {
     return itemDAO.findById(itemId);
   }
-  
+
   /**
    * Update item
    *
-   * @param item item
    * @param title title
    * @param description description
    * @param category category
+   * @param visibilityLimited visibilityLimited
+   * @param location location
    * @param slug slug
    * @param expiresAt expiresAt
    * @param unitPrice unitPrice
    * @param priceCurrency priceCurrency
    * @param amount amount
    * @param unit unit
-   * @param visibilityLimited visibility limited
-   * @param sellerId seller id
-   * @param soldAmount sold amount
+   * @param soldAmount soldAmount
+   * @param allowPurchaseContactSeller allowPurchaseContactSeller
+   * @param allowPurchaseCreditCard allowPurchaseCreditCard
+   * @param deliveryTime deliveryTime
+   * @param contactEmail contactEmail
+   * @param contactPhone contactPhone
+   * @param termsOfDelivery termsOfDelivery
+   * @param sellerId sellerId
    * @param modifier modifier
-   * 
    * @return updated item
    */
-  @SuppressWarnings ("squid:S00107")
-  public Item updateItem(Item item, LocalizedEntry title, LocalizedEntry description, Category category, Location location, String slug, OffsetDateTime expiresAt, String unitPrice, Currency priceCurrency, Long amount, String unit, boolean visibilityLimited, UUID sellerId, Long soldAmount, UUID modifier) {
+  public Item updateItem(Item item, LocalizedEntry title, LocalizedEntry description, Category category, boolean visibilityLimited, Location location, String slug, OffsetDateTime expiresAt, String unitPrice, Currency priceCurrency, Long amount, String unit, Long soldAmount, Boolean allowPurchaseContactSeller, Boolean allowPurchaseCreditCard, Integer deliveryTime, String contactEmail, String contactPhone, String termsOfDelivery, UUID sellerId, UUID modifier) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Method has 21 parameters, which is greater than 7 authorized. ([squid:S00107](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AS00107))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.dcfb-api%3Adcfb-api&pullRequest=36&issues=AWXX0RqA_eEZPtUmBuqe&open=AWXX0RqA_eEZPtUmBuqe)

[ISSUE_KEY]: AWXX0RqA_eEZPtUmBuqe